### DISCUSSION TITLE
Are there cases where a process-exit can be triggered with error code 71

### DISCUSSION THREAD

**Discussion by nicknotfun** [uNetworking/uWebSockets.js#940] [Q&A] [answered]
We've deployed uWebSockets.js in production and it works great- however exceedingly infrequently (and we haven't yet narrowed the cause) our process exists with code 71.

This is a Kubernetes environment and as far as we're aware we're capturing all log output (and there is none that is relevant). So came here to ask if there are known circumstances where this can occur? Perhaps we neglected to register some error callback?

We are a non-SSL App with a root HTTP handlers [returns hello world], as well as a websocket behaviour handler - no other uSocket HTTP handling. It is also one of many processes using the same codebase - but the only one dealing with websockets.

**Comment by uNetworkingAB** [comment, marked as answer]
#define EX_OSERR        71      /* system error (e.g., can't fork) */

No I don't think so. We can return 139 from std::terminate() but that should be obvious since we print an error in that case.

**Reply by nicknotfun** [reply]
Thank you, our investigation continues. 

For an update on our end:
We do think it might indeed be related to a parent-wrapper process in our runtime (which does use fork) ; but mostly it hiding the true exit reason (potentially OOM).
